sqlx-postgres/src/connection/stream.rs

```diff
@@ -53,13 +53,14 @@ macro_rules! read_packet {
         loop {
             read_packet!($(@$blocking)? @stream $self, 0, 5);
 
-            let mut header: Bytes = $self.stream.take(5);
+            // peek at the messaage type and payload size
+            let r#type = MessageType::try_from(*$self.stream.get(0,1))?;
```

Comment:
Try and keep to rustfmt even though its not helping within the macro.

```rust
(*$self.stream.get(0, 1))?;
```

Reply:
I added the additional parens.  I reran rustfmt +nightly with the project rustfmt.toml, however I'm not sure this is the desired fix.

Reply:
Oh. Maybe I should have been more explicit. I meant the lack of spacing after the `,` in `0,1`.

Here is how rustfmt would format this if it was outside of the macro:

```rust
let r#type = MessageType::try_from(*$self.stream.get(0, 1))?;
```

Reply:
Ah, thanks for clarifying!  I hadn't realized macro calls were treated differently.